sql/testdata/update

```diff
@@ -267,3 +267,28 @@ INSERT INTO pks VALUES (1, 2, 3), (4, 5, 3)
 
 statement error duplicate key value \(k2,v\)=\(5,3\) violates unique constraint "i"
 UPDATE pks SET k2 = 5 where k1 = 1
+
+# Test updating only one of the columns of a multi-column primary key.
+
+statement ok
+CREATE TABLE ab (
```

Comment:
This could reuse table `pks` from line 257, which has a multi column primary key